data/json/items/gun/shot.json

```
@@ -160,7 +160,7 @@
     "copy-from": "mossberg_500",
     "type": "GUN",
     "name": { "str": "Mossberg 500 Security", "str_pl": "Mossberg 500 Security" },
-    "description": "The Mossberg 500 is a popular series of pump-action shotguns, often acquired for military use.  It is noted for its high durability and low recoil.  This one is fitted with an 18.5 inch barrel.",
+    "description": "The Mossberg 500 is a popular series of pump-action shotguns, often acquired for military use.  It is noted for its high durability and low recoil.  This one is fitted with an 18.5-inch barrel.",
```

```suggestion
    "description": "The Mossberg 500 is a popular series of pump-action shotguns, often acquired for military use.  It is noted for its high durability and low recoil.  This one is fitted with an 18.5 inch barrel.",
```

There are a number of entries below where "## unit" was changed to "##-unit". This is debatably correct at best and I don't think it should be changed.